core/domain/suggestion_services_test.py

```diff
@@ -1648,3 +1648,548 @@ def test_get_voiceover_application_from_model_with_invalid_type_raise_error(
             'Invalid target type for voiceover application: invalid_type'):
             suggestion_services.get_voiceover_application(
                 self.voiceover_application_model.id)
+
+
+class CommunityContributionStatsUnitTests(test_utils.GenericTestBase):
+    """Test the functionality related to updating the community contribution
+    stats.
+
+    Note: testing whether resubmiting a rejected translation increases the
+    translation suggestion count could not be tested because the
+    pre_update_validate method in suggestion_registry.py is not implemented for
+    translation suggestions. As a result, rejected translation suggestions
+    cannot be resubmitted for review.
```

Comment:
Is this an intentional product choice? It slightly confuses me whether this is actually what we want. Are we planning on changing this? Is there a TODO issue that can be referenced here to make sure this test coverage is re-added if we do plan to eventually support that scenario?

Reply:
@kaylahardie can you file an issue to track this, and add a TODO here to either remove the TODO, or add the missing test once the functionality is added (based on the resolution of the product question)? I'd rather not block your PR given the time frame it needs to be submitted by, but I don't want this particular item to be forgotten.

Reply:
Filed an issue and added a TODO